lib/ansible/modules/source_control/gitlab_tag.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/python
+# (c) 2016, Werner Dijkerman (ikben@werner-dijkerman.nl)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: gitlab_tag
+short_description: Creates/deletes tags on Gitlab projects
```

Comment:
This is a nitpick, I'm not 100% sure if it's right, but it looks like GitLab like to go by "GitLab" and not "Gitlab"